sys/include/net/netopt.h

```diff
@@ -240,6 +240,29 @@ typedef enum {
      */
     NETOPT_RF_TESTMODE,
 
+    /**
+     * @brief LoRA options.
+     */
+    NETOPT_LORA_BANDWIDTH,                        /**< bandwidth of LoRa */
+    NETOPT_LORA_SPREADING_FACTOR,                 /**< spreading factor of LoRa */
+    NETOPT_LORA_CODING_RATE,                      /**< coding rate of LoRa */
+    NETOPT_LORA_SINGLE_RECEIVE,                   /**< when enabled, RX is turned of upon reception of a packet */
+    NETOPT_LORA_RX_TIMEOUT,                       /**< Reception timeout of LoRa packet */
+    NETOPT_LORA_TX_TIMEOUT,                       /**< Reception timeout of LoRa packet */
+    NETOPT_LORA_SYMBOL_TIMEOUT,                   /**< Symbol timeout */
+    NETOPT_LORA_SYNCWORD,                         /**< Syncword */
+    NETOPT_LORA_HOP,                              /**< Frequency hopping enabled */
+    NETOPT_LORA_HOP_PERIOD,                       /**< Frequency hopping period */
+    NETOPT_CRC,                                   /**< CRC check enabled */
+    NETOPT_LORA_IMPLICIT,                         /**< Implicit mode enabled */
+    NETOPT_LORA_IQ_INVERT,                        /**< IQ inverted enabled */
+    NETOPT_LORA_PAYLOAD_LENGTH,                   /**< Payload length */
+    NETOPT_LORA_PREAMBLE_LENGTH,                  /**< Preamble length */
+    NETOPT_LORA_MODE,                             /**< LoRa mode */
+    NETOPT_LORA_MAX_PAYLOAD,                      /**< Maximum payload size */
+    NETOPT_LORA_RANDOM,                           /**< Random value */
+    NETOPT_LORA_TIME_ON_AIR,                      /**< Time on air */
+
```

Comment:
I don't have a solution at hand but I'm wondering if we need/want all these options as generic netopts.